src/components/Auth/ChangePassword.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+import { useState, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { validatePassword } from '../Form/Validators'
+import client from '@/database/client'
+import ValidatedInput from '../Form/ValidatedInput'
+
+export default function ChangePassword() {
+    const [password, setPassword] = useState<string>('')
+    const [formValid, setValid] = useState<boolean>(false)
```

Comment:
I think a common pattern used with useState return values is [variableName, setVariableName]. Having them the same helps identify what state dispatcher relates to which state